README.md

```diff
@@ -5,17 +5,25 @@ Dart package for the Open Food Facts API.
 
 Easily access to more than 1.3 million products from all around the world.
 Open Food Facts is powered by global contributors and is constantly growing thanks to them.
-
+The database in under the OdBL. This means attributing the source and also contributing back any additions, which this packages makes easy to do.
 You can check the terms of use here : [Terms of use](https://world.openfoodfacts.org/terms-of-use).
 
 ## List of current features
 TODO: complete me
 
 ### READ
```

Comment:
Maybe check with @PrimaelQuemerais but I think that most of these can be checked off
- Product: you can retrieve information about products
- Product images: you can fetch product images
- Send images: that you can do

Not sure about the remaining ones.